### ISSUE TITLE
Development Request - integrate Sprite Editor to Vchar64

### ISSUE THREAD

**Issue by alexgoldblat** [ricardoquesada/vchar64#31]
Ricardo i love your tool , it would be awesome if you make a different tool or in-house tool (integrated) to Vchar 64 option to edit sprites (MC and Single color) , i think you already have the knowledge to make a good support for it and the editing could be the same like Chars but instead of export to Char , it will be for sprites. later you can do the import/export from Sprites to 3 colors Chars and the reverse features. it could be with the same abilities like on Vchar64 but now for Sprites .

**Comment by ricardoquesada** [comment]
yeah, there are no sprite editors for mac/linux, so having one would be nice.
although I don't think I'll have the time in 2017 to do it... perhaps somewhere in 2018 (or later).

**Comment by Esshahn** [comment]
+1 for this one

**Comment by dobkeratops** [comment]
would this ideally include the ability to place sprites in maps to make complete screens'?

**Comment by ricardoquesada** [comment]
@dobkeratops that would be a nice addition. One day, when I find time, I'll implement it :)
